.vscode/extensions.json

```diff
@@ -0,0 +1,7 @@
+{
+    "recommendations": [
+        "ms-vscode.cmake-tools",
```

Comment:
Shouldn't we add "SynthWave '84" them with glow enabled to the list of recommendations? (jk)

Reply:
We should recommend the uncrustify extension, though. Will add.

Reply:
> Shouldn't we add "SynthWave '84" theme with glow enabled to the list of recommendations? (jk)

I don't know. Does it work well with Doki-Theme?

Reply:
Unfortunately not :(